SimpleMessager/MessageApp/src/main/java/org/example/ClientHandler.java

```diff
@@ -0,0 +1,215 @@
+                        break;
+                    case "LIST": //klasör
+                        handleList();
+                        break;
+                    case "READ":
+                        handleRead(tokens);
+                        break;
+                    case "QUIT":
+                        handleQuit();
+                        break;
+                    case "REGISTER":
+                        handleRegister(tokens);
+                        break;
+                    default:
+                        out.println("UNKNOWN COMMAND");
+                        break;
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        } finally {
+            try {
+                socket.close();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private void handleQuit() {
+        out.println("QUIT SUCCESS");
+    }
+
+    private void handleRead(String[] tokens) {
+        String messageID = tokens[1];
+        String readMessage="";
+        if(person.getAuthenticatedUser()!=null){
+                for(Message msg : Message.getMessageList()){
+                    if(msg.messageId.equals(messageID)){
+                        messageReadView.add("Gönderici: "+msg.sender);
+                        messageReadView.add("Alıcı : "+msg.reciever);
+                        messageReadView.add("Konu : "+msg.subtopic);
+                        messageReadView.add("İçerik : "+msg.content);
+                    }
+                }
+            }
+        readMessage=gson.toJson(messageReadView);
+        out.println(readMessage);
+        messageReadView.clear();
+
+        }
+
+    private void handleList() {
+        String reciveJSON="";
+        String sendJSON="";
+        String messageSender = person.getAuthenticatedUser();
+            if (person.getAuthenticatedUser().equals(messageSender)) {
+                for (Message msg : Message.getMessageList()) {
+                    if (messageSender.equals(msg.sender)) {
+                        message=msg.senderFile.get(messageSender);
+                        message.sendLıst.add("Gönderilen Kutusu -----> ");
+                        message.sendLıst.add("Message Id: " + message.messageId);
+                        message.sendLıst.add("Gönderen : "+message.sender);
+                        message.sendLıst.add("Alıcı : " + message.reciever);
+                        message.sendLıst.add("Konu : "+message.subtopic);
+                        sendLıstView.add(message.sendLıst);
+                        sendJSON =gson.toJson(sendLıstView);
+
+                    }
+                        if(person.getUserID().contains(msg.reciever.substring(0,3) + person.getUserCredentials().get(msg.reciever).substring(0,3))){
+                            message=message.reciverFile.get(msg.reciever);
+                            message.reciveLıst.add("Gelen Kutusu ----> ");

```

Comment:
Gönderilenler ve gelenler dışında bir klasördeki mesajların listelenmesini istesek ne olacak?
Sistemdeki her yeni klasör için kodu değiştiremeyiz değil mi?